README.md

```diff
@@ -65,13 +67,62 @@ The GOV.UK frontend toolkit dependencies are listed at the top of the `_govuk_el
     // @import "url-helpers";                         // Function to output image-url, or prefixed path (Rails and Compass only)
 
 
-Either copy these files your Sass directory, or configure the `includeFiles` path if you’re using a task runner like Grunt.
+Add the `node_modules` directory to the `includePaths` property of your Sass plugin - if you're using a task runner like Gulp, to reference the location of these files.
```

Comment:
Should say `node_modules/govuk-elements-sass` directory to match the example below I think.